README.md

```diff
@@ -1,63 +1,163 @@
 # npl-integrations
 
-## Pre-requisites
-Make sure your `hosts` file includes keycloak at 127.0.0.1:
+The npl-integrations repo contains a sample project that demonstrates how to integrate with the NPL engine using
+different programming languages and frameworks.
+It includes a Python listener service, a typescript React webapp, and a python Streamlit UI.
 
-![img.png](docs/img.png)
+The underlying NPL code is an extension of the IOU, which can be found
+in the [npl-starter](https://github.com/NoumenaDigital/npl-starter) repository.
 
 ## Building and running with docker locally
-1. Run `mvn clean install` to build and generate NPL-api and clients.
-2. Run `docker compose up --build -d` to ensure python and frontend are also build, then create and start containers.
 
-URLs:
-* Swagger UI of Engine APIs: http://localhost:12000/swagger-ui/
-* Keycloak admin console: http://localhost:11000
-* Webapp: http://localhost:8090
+The first deployment possibility consists of running the NPL code and all services locally.
```

Comment:
I don't think there's really any conceptual difference between deploying to a prod environment and deploying locally, so I would rephrase this to just say that we deploy to an Engine container, which you _can_ run locally.